apcd-cms/src/apps/view_users/templates/view_user_modal.html

```diff
@@ -26,7 +26,7 @@ <h4 class="modal-title">Details for User: {{r.user_name}} ({{r.user_id}})</h4>
                 <dt class="c-data-list__key">Email</dt>
                 <dd class="c-data-list__value">{{r.user_email}}</dd>
                 <dt class="c-data-list__key">Organization</dt>
```

Comment:
```suggestion
                <dt class="c-data-list__key">Entity Organization</dt>
```
To match the other admin tables and to be clear which field this is pulling

Reply:
Changed name field title to Entity Organization and added the field to the edit modal.